src/main/resources/site/parts/page-heading-with-menu/page-heading-with-menu.js

```diff
@@ -35,22 +36,24 @@ function handleGet(req) {
         menuItems = menuItems[menuItems.findIndex(function (value) {
             return value.name === language;
         })];
-        var frontPageUrl = libs.portal.pageUrl({id: site._id});
+
+        // Må ha tre separate kall på pageUrl for å sikre korrekt url (caches)
+        //TODO: Fjerne eksplesitt https når dette er løst på BigIP
         var languageSelectors = [
             {
-                href: frontPageUrl + '/no',
+                href: libs.portal.pageUrl({type: "absolute", path: "/www.nav.no/no"}).replace("http:", "https:"),
                 title: 'Norsk (Globalt språkvalg)',
                 text: 'Norsk',
-                active: (language || language === 'no' ? 'active' : '')
+                active: (language === 'no' ? 'active' : '')
```

Comment:
```suggestion
                active: (!language || language === 'no' ? 'active' : '')
```
Har alt innhold language prop?